x-pack/plugins/monitoring/public/components/apm/instance/instance.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { MonitoringTimeseriesContainer } from '../../chart';
+import { EuiFlexGrid, EuiFlexItem, EuiSpacer, EuiPage, EuiPageBody } from '@elastic/eui';
```

Comment:
Does this mean the apm monitoring part will be built on top of EUI and already "fulfill" all accessibility criterias?

Reply:
Unfortunately, I'm my experience this isn't always the case. We do some accessibility wins for free by moving to EUI but it doesn't solve the entire problem. It feels like we'll always need to be aware of this and proactively accounting for it.